### PR TITLE
[Doc] [Security] Added note about required Admin for "sonata_media_download" route

### PR DESCRIPTION
**Pull request by phansys**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets |
| License       | MIT
| Doc PR        |

Added note about required Admin for ```sonata_mediadownload``` route.

### CODE REVIEW

File: Resources/doc/reference/security.rst [hunk 1]
```diff
@@ -56,6 +56,10 @@ The related download route name is ``sonata_media_download``.
 
     <a href="{{ path('sonata_media_download', {'id': media|sonata_urlsafeid }) }}">Download file</a>
 
+.. note::
+
+    You must have an admin registered for the media object in order to work with this route.
```
Comment (rande) [inline Resources/doc/reference/security.rst]: you should add ``and this strategy``

Comment (phansys) [inline Resources/doc/reference/security.rst]: Good point @rande. But this is not needed for other secured strategies such as
```sonata.media.security.forbidden_strategy``` and ```sonata.media.security.connected_strategy```?

Comment (rande) [inline Resources/doc/reference/security.rst]: @phansys not it is not required

Comment (phansys) [inline Resources/doc/reference/security.rst]: So, what route might I use to get a secured path?
In my use case, I have this config:
```yaml
# config.yml
sonata_media:
    contexts:
        content:
            providers:
                - sonata.media.provider.file
            download:
                strategy: sonata.media.security.connected_strategy
                mode: http
```
So in view I added this route, but it complains about the admin, so I had to configure one for my Media object. Notice that my strategy isn't ```sonata.media.security.superadmin_strategy```.

**Comment by Soullivaneuh** [comment]
According to the new [Sonata version management](https://github.com/sonata-project/SonataAdminBundle/issues/3053) and next major [release plan](https://github.com/sonata-project/SonataAdminBundle/issues/3731), this project has been refactored regarding branching and versioning.

If you see this message, your PR concerns a patch or a minor release and is not targeting the right branch.

So I'm closing this one, but don't see it as a refusal. If you think your work is still relevant and want to continue, feel free to reopen it on the right branch (e.g. the default one).

Regards.

**Pull request closed without merging**